scanner/src/routes/q/[iss]/[product]/[id]/+page.svelte

```diff
@@ -24,27 +29,45 @@
     } else {
       console.error("Fetching metadata failed", result)
       meta = undefined
+      error = result.data.detail[0].msg
+
+      setTimeout(() => {
+        rescan()
+      }, 2000)
     }
     loading = false
   }
+
+  function rescan() {
+    goto("/scan")
+  }
+
   onMount(() => {
     load()
   })
 </script>
 
-{#if loading || !meta}
+{#if loading}
   <div class="loader">
     <Loading type="light" />
   </div>
-{:else}
+{:else if meta}
   <div class="content">
     <Header logoUrl={meta.logo_url} />
-    <BasicInformation {meta} product={$page.params.product} />
+    <BasicInformation {meta} product={$page.params.product} {verified} />
     {#each meta.supported_dataproducts as dataProduct}
       <DataProduct productBrief={dataProduct} />
     {/each}
   </div>
   <Footer />
+{:else}
+  <div class="relative failed-verification-wrapper">
+    <img src={ErrorSvg} alt="" aria-hidden="true" />
+    <p>{error}</p>
+  </div>
+  <div class="actions-wrapper">
+    <p>Redirecting to Scan page...</p>
```

Comment:
```suggestion
    <p>Returning to scanning mode...</p>
```
End-users don't care about "redirects" or "scan pages"